src/rules/no-private-properties.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * @fileoverview Detects usages of "non-public" properties
+ * @author Michael Stramel <https://github.com/stramel>
+ */
+
+import {Rule} from 'eslint';
+import * as ESTree from 'estree';
+import {TemplateAnalyzer} from '../template-analyzer';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      description: 'Detects usages of "non-public" properties',
+      category: 'Best Practices',
+      url:
+        'https://github.com/43081j/eslint-plugin-lit/blob/master/docs/rules/no-private-properties.md'
+    },
+    messages: {
+      unsupported: 'TODO'
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          prefix: {type: 'string'}
+        },
+        additionalProperties: false
+      }
+    ]
+  },
+
+  create(context): Rule.RuleListener {
+    // variables should be defined here
+    const options = {
+      prefix: '_',
```

Comment:
i'd rather this _always_ be a regex and just `new RegExp(prefix)` it somewhere here and re-use that (so the default is `^_`)

Reply:
Yeah, going to change to follow the `"@typescript-eslint/member-naming"` with options:

```
      {
        "private": "^__",
        "protected": "^_"
      }
```